pytensor/tensor/rewriting/linalg.py

```diff
@@ -310,3 +320,65 @@ def local_log_prod_sqr(fgraph, node):
 
         # TODO: have a reduction like prod and sum that simply
         # returns the sign of the prod multiplication.
+
+
+def local_inv_kron_to_kron_inv(fgraph, node):
```

Comment:
Add these as issues instead?

Reply:
I think we actually ended up rolling them all into a single rewrite, we can just delete these